apps/ima/index.js

```diff
@@ -33,9 +33,274 @@ module.exports = {
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-eta') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/entry-clearance',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-visa') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/travel-ban',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('travel-ban') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        }
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/arrival-date'
+    },
+    '/travel-ban': {
+      behaviours: SaveFormSession,
+      fields: [
+        'entered-with-travel-ban',
+        'entered-with-travel-ban-detail'
+      ],
+      forks: [
+        {
+          target: '/electronic-travel-authorisation',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-eta') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/entry-clearance',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-visa') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        }
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/arrival-date'
+    },
+    '/entry-clearance': {
+      behaviours: SaveFormSession,
+      fields: [
+        'arrived-without-clearance',
+        'arrived-without-clearance-detail'
+      ],
+      forks: [
+        {
+          target: '/electronic-travel-authorisation',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-eta') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        }
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/arrival-date'
+    },
+    '/electronic-travel-authorisation': {
+      behaviours: SaveFormSession,
+      fields: [
+        'without-eta',
+        'without-eta-detail'
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/arrival-date'
+    },
+    '/arrival-date': {
+      next: '/confirm'
+    },
```

Comment:
I remember the figma not having too many different paths so I'm wondering why there are so many forks @mislam987 could you show me how the user navigates through the form.  Slack me when you're free